tests/models/spatialview_model_tests.py

```diff
@@ -283,6 +283,16 @@ def test_spatial_view_isactive_set_to_false_removes_views(self):
 
         spatialview.delete()
 
+    def test_spatial_view_without_geometrynode(self):
+        spatialview = self.generate_valid_spatiatview()
+        spatialview.geometrynode = None
+        with self.assertRaises(Exception):
+            spatialview.full_clean()
+            spatialview.save()
+
+        with self.assertRaises(SpatialView.DoesNotExist):
+            fetched_spatialview = SpatialView.objects.get(pk=spatialview.spatialviewid)
+
```

Comment:
I would just chop the save() and the last assertion and let the test fail naturally if the validation doesn't fire. We don't need to test the test helper itself.
```suggestion
```